playground/frontend/lib/pages/standalone_playground/widgets/more_actions.dart

```diff
@@ -68,9 +68,10 @@ class _MoreActionsState extends State<MoreActions> {
               title: Text(appLocale.shortcuts),
               onTap: () {
                 AnalyticsService.get(context).trackOpenShortcutsModal();
-                showDialog<void>(
+                BeamDialog.show(
```

Comment:
Make a list with buttons.
A CloseButton button likely must take at most a single parameter with a callback to close. See if `PublicNotifier` can be reused here.